modules/core/css/lib/variables/colors.scss

```diff
@@ -4,8 +4,9 @@
 $wdc-color-error: $wdc-color-cinnamon-500;
 $wdc-color-error-light: $wdc-color-cinnamon-200;
 $wdc-color-error-dark: $wdc-color-cinnamon-600;
-$wdc-color-alert: $wdc-color-cantaloupe-400;
-$wdc-color-alert-light: $wdc-color-cantaloupe-200;
-$wdc-color-alert-dark: $wdc-color-cantaloupe-500;
+$wdc-color-alert: $wdc-color-cantaloupe-500;
+$wdc-color-alert-light: $wdc-color-cantaloupe-400;
+$wdc-color-alert-dark: $wdc-color-cantaloupe-600;
 $wdc-color-shadow-light: rgba(82, 97, 115, 0.14);
 $wdc-color-shadow: rgba(82, 97, 115, 0.18);
+$wdc-color-input-focus-border: $wdc-color-blueberry-400;
```

Comment:
This shouldn't get added here if you need it, it would go in design-assets. I also think it already exists as `$wdc-color-input-focus-outline`